config/initializers/high_voltage.rb

```diff
@@ -0,0 +1,3 @@
+HighVoltage.configure do |config|
+  config.routes = false
```

Comment:
Haven't seen `HighVoltage` before. I'm guessing you're removing the default routes here because we are using our own customised routes for the static pages, right? (the ones created on the `routes.rb` below I mean)

Reply:
Yeah, that's right - we need to change the Routes to point to out custom controller now